app/components/ui/molecules/SearchBox.test.js

```diff
@@ -54,30 +54,6 @@ describe('SearchBox component tests', () => {
     expect(wrapper.find('[data-test-id="search-icon"]')).toHaveLength(1)
   })
 
-  it.each([[1, 'option'], [2, 'f'], [3, 'xyz'], [4, '']])(
```

Comment:
I would just skip this since we are going to go back at having an autosuggest in the future.